service/src/main/kotlin/fi/espoo/evaka/invoicing/client/S3DocumentClient.kt

```diff
@@ -58,13 +58,20 @@ class S3DocumentClient(
         return documentKey
     }
 
+    fun getPdf(bucket: String, key: String): ByteArray {
+        val s3Object = s3Client.getObject(bucket, key)
+        return IOUtils.toByteArray(s3Object.objectContent)
```

Comment:
Could maybe use Kotlin builtin `s3Object.objectContent.readAllBytes()`? IOUtils seems like something more intended for AWS SDK internal use even though it's public